Language/Haskell/GhcMod/Gap.hs

```diff
@@ -96,7 +96,12 @@ import TcRnTypes
 #endif
 
 #if __GLASGOW_HASKELL__ >= 706
+#if defined(MIN_VERSION_GLASGOW_HASKELL) && MIN_VERSION_GLASGOW_HASKELL(8,0,2,0)
```

Comment:
Oh one thing `#if __GLASGOW_HASKELL__ >= 802` is better since it's defined by GHC itself not by cabal.